### ISSUE TITLE
Table exports with a long line in PDFs

### ISSUE THREAD

**Issue by priyatharsan** [plotly/plotly.py#1280]
related to: https://github.com/plotly/orca/issues/147 and https://github.com/plotly/streambed/issues/11618

The PDF exports display line in the following environments:
- Windows 10 / Chrome viewer
- Windows 10 / Adobe-reader
- MacOS10.13 / Chrome70

Example : 
```
import plotly.plotly as py
import plotly.graph_objs as go

import pandas as pd
import colorlover as cl

colors = cl.scales['5']['seq']['Blues']
data = {'Year' : [2010, 2011, 2012, 2013, 2014],
        'Color' : colors}
df = pd.DataFrame(data)


trace0 = go.Table(
  header = dict(
    values = ["Color", "<b>YEAR</b>"],
    line = dict(color = 'white'),
    fill = dict(color = 'white'),
    align = ['center'],
    font = dict(color = 'black', size = 12)
  ),
  cells = dict(
    values = [df.Color, df.Year],
    line = dict(color = [df.Color]),
    fill = dict(color = [df.Color]),
    align = 'center',
    font = dict(color = 'black', size = 11)
    ))

data = [trace0]

py.iplot(data, filename = "row variable color")
```

![capture](https://user-images.githubusercontent.com/25846124/48807653-362a9f00-ecec-11e8-80a3-c8f38ebbaa1e.PNG)

Please note that this behavior is happening only when saving via Python API call and exporting via the plot's view page at [plot.ly](https://plot.ly/organize/home). I don't see the line when saving/exporting via Chart studio.

**Comment by priyatharsan** [comment]
Apologies for not being clear. I'm exporting from the plot's view page:https://plot.ly/~priyatharsan/977/

![screenshot 21](https://user-images.githubusercontent.com/25846124/48850559-f5c43300-ed76-11e8-979c-19214030c73e.png)

**Comment by jonmmease** [comment]
No problem.  So at that point, the figure has moved out of plotly.py and is being stored in JSON form on chart studio cloud.   Could you try to identify a difference in the JSON representation between the figure that exports properly and the one that doesn't?

**Comment by priyatharsan** [comment]
Here is the plot that exports with a line: https://plot.ly/~priyatharsan/977/
and no line: https://plot.ly/~priyatharsan/986/
The only differences I can find are:
 `cells.fill.colorsrc`, `cells.line.colorsrc`, `cells.valuessrc`, `header.alignsrc`, `header.alignsrc`, and `uid`

**Comment by etpinard** [comment]
@priyatharsan I'll close this ticket

The latest version of orca, plotly.py and plotly.js are able to export that `table` graph w/o the erroneous line

**Comment by priyatharsan** [comment]
>Do you see a line here?
@jonmmease  No I don't any line. 
>this version is not relevant if you're exporting to a PDF through chart studio.
Apologies for the confusion and it makes more sense now as when I tried with plotly.py's `plotly.io.write_image` I don't see any line so this has something to do with the Chart Studio. Thank you for clarifying this.

@etpinard Thank you for the update I will follow-up with issue's status at https://github.com/plotly/streambed/issues/11618

**Comment by vishalsharma3003** [comment]
I had the same problem and there is not much information provided. I don't know the exact reason why the erroneous line is there in these graphs but I have a workaround for this.
Try setting the background color of the graph and you'll notice that the line is aligned with the background so I adjusted the margins and put the right margin to zero and the problem was resolved.

layout(margin = list(r = 0, pad = 0)) 

Before:
![image](https://user-images.githubusercontent.com/19642193/75415249-15baad00-5951-11ea-9009-2f409acebc63.png)
 
After:
![image](https://user-images.githubusercontent.com/19642193/75415274-23703280-5951-11ea-803e-63ad729dbc20.png)
